doc/library-main.dox

```diff
@@ -0,0 +1,11 @@
+/*!
+@mainpage Google Cloud Platform C++ Client Libraries
+
+This page contains the reference guide to the Google Cloud Platform C++ Client
+Libraries.
+
+## Alpha Libraries
+
+- @ref bigtable-main "Google Cloud Bigtable"
```

Comment:
I think Firestore and Storage are incomplete, and hence not yet qualified to be labelled "Alpha".

Reply:
Ah okay. It would help to label what they are as they currently exist in the namespace docs and index. "In development" fits for the case, but based on other client libraries, WIP client libraries are usually not merged into master until ready and kept in a branch instead.

Reply:
Done.